cfme/utils/ssh.py

```diff
@@ -309,18 +306,34 @@ def run_command(
             stderr = session.makefile_stderr()
             while True:
```

Comment:
Might be good to have a comment on why we are doing it this way, so no one unwittingly simplifies it later.